examples/forecasting/advanced_forecasting_example.py

```diff
@@ -0,0 +1,70 @@
+                                                                         target_column="ConfirmedCases",
+                                                                         time_column="Date",
+                                                                         # hyperparameter_tune_kwargs={
+                                                                         #     'scheduler': 'local',
+                                                                         #     'searcher': 'random',
+                                                                         #     "num_trials": 2
+                                                                         # },
+                                                                         hyperparameter_tune_kwargs="auto",
+                                                                         quantiles=[0.1, 0.5, 0.9],
+                                                                         refit_full=True,
+                                                                         hyperparameters={
+                                                                             "MQCNN": {
+                                                                                 'context_length': ag.Int(70, 90,
+                                                                                                          default=prediction_length * 4),
+                                                                                 "num_batches_per_epoch": 10,
+                                                                                 "epochs": 5},
+                                                                             "DeepAR": {
+                                                                                 'context_length': ag.Int(70, 90,
+                                                                                                          default=prediction_length * 4),
+                                                                                 "num_batches_per_epoch": 10,
+                                                                                 "epochs": 5},
+                                                                             "SFF": {
+                                                                                 'context_length': ag.Int(70, 90,
+                                                                                                          default=prediction_length * 4),
+                                                                                 "num_batches_per_epoch": 10,
+                                                                                 "epochs": 30},
+                                                                         },
+                                                                         time_limit=60
+                                                                         )
+
+
+predictor = None
```

Comment:
> Why do we need to initially set it to `None`

Setting it to None and then do the loading so that we can make sure the loading function is working properly.